src/elements/FeaturesTablePanel/featuresTablePanel.scss

```diff
@@ -0,0 +1,131 @@
+@import '../../scss/colors';
+@import '../../scss/borders';
+
+.features-panel {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 500px;
+  height: 100%;
+
+  .features-panel__content {
+    padding: 10px 15px;
+  }
+
+  .features-panel__header {
+    padding: 5px 15px;
+  }
+
+  .features-panel__header-project {
+    margin-bottom: 5px;
+    color: $topaz;
+    font-weight: bold;
+    font-size: 14px;
+  }
+
+  .features-panel__header-vector {
+    display: flex;
+    justify-content: space-between;
+
+    .features-panel__header-vector-name {
+      display: flex;
+      align-items: center;
```

Comment:
Please change to:
```suggestion
      align-items: baseline;
```
So the text is aligned like this:
![image](https://user-images.githubusercontent.com/13918850/115991578-d7ab2e80-a5d1-11eb-8279-4c3f0178ccdb.png)
Instead of this:
![image](https://user-images.githubusercontent.com/13918850/115991587-dc6fe280-a5d1-11eb-915e-62772cdd6a9a.png)

When vertically aligning flex items that have text, most times aligning by `baseline` would give the desired result. (Like with everything else: not _always_, but most times.)